### DISCUSSION TITLE
Search results web part template - how to add the Info/Details ability to each row

### DISCUSSION THREAD

**Discussion by jbrenny** [microsoft-search/pnp-modern-search-layouts#6] [Q&A]
I have a successful PNP Search Results page BUT can't figure out one feature. Here's an example.

A search results returns 5 documents. One of the documents needs have a site column value changed (like Status from PENDING to COMPLETED).  I am not updating/editing the document but a site column value. How can I get to the DocInfo/Details link to modify a site column value? I've enabled the SELECT feature. I've tried selecting the row & redirecting to another page where I can perform the update but I can't figure out how to pass parameters from the search results to another page with only the ONE document that I can then update the meta data.  

Is this possible?

**Comment by kasperbolarsen** [comment]
Let me see if I understand the question.  Are you look for a way to add a button or similar to the layout that will open the "edit details form" for the current result item?
